config/permissions.tmpl.yml

```diff
@@ -14,6 +14,8 @@
 - :svc
 - :vi
 - ems-type:ansible_tower_configuration
+- ems-type:atomic
+- ems-type:atomic_enterprise
```

Comment:
@durandom @yaacov should we really reintroduce these? Can't we just keep these out?